edith-step/src/UserChart.js

```diff
@@ -163,8 +164,22 @@ const PieChart = () => {
         console.log(response);
         let itemNames = [];
         let itemValues = [];  
-        const items = JSON.parse(response.data.items);
-        console.log(items);
+        let items = [];
+        const itemsList = JSON.parse(response.data.items);
+        if (props.dateSelection != "") {
+          console.log(props.dateSelection);
```

Comment:
Do we need this log?

Reply:
No, I just had this intermittently for error checking.